src/generator/apitypes.ts

```diff
@@ -80,10 +80,10 @@ class TypesGenerator implements Generatable<TypesGeneratorStructure> {
 
   toHTML() {
     return `<div>
-    <h1 class="text-3xl" id="types">Types</h1>
+    <h1 class="text-3xl dark:text-white" id="types">Types</h1>
         <div>
           <div class="ml-4">
-            <h3 class="mb-2 text-2xl font-normal" id="input-types">Input Types</h3>
+            <h3 class="mb-2 text-2xl font-normal" id="input-types dark:text-white">Input Types</h3>
```

Comment:
Please do the same for output types on line 94. Right now it is of wrong color in dark mode